onediff_comfy_nodes/prestartup_scripts/npu.py

```
@@ -0,0 +1,59 @@
+_IS_NPU_AVAILABLE = False
+try:
+    import torch_npu
+    from torch_npu.contrib import transfer_to_npu
```

Ruff

`torch_npu.contrib.transfer_to_npu` imported but unused

Remove unused import: `torch_npu.contrib.transfer_to_npu`

(F401)